nexong/models.py

```diff
@@ -194,7 +194,34 @@ class Educator(models.Model):
     birthdate = models.DateField(null=True)
 
 
+class UserManager(BaseUserManager):
+    def create_user(self, email, password=None):
+        if not email:
+            raise ValueError("No email provided")
+        if not password:
+            raise ValueError("No password provided")
+
+        user = self.model(email=self.normalize_email(email))
+        user.set_password(password)
+        user.is_admin = False
+        user.is_staff = False
+        user.save(using=self._db)
+        return user
+
+    def create_superuser(self, email, password=None):
+        user = self.create_user(email, password=password)
+        user.is_admin = True
```

Comment:
At the moment, for me says:
`dajngo.db.utils.ProgrammingError: Column nexong_user.is_admin does not exist`
Probably the problem is related to the line highlighted.

Reply:
That is strange, one of the latest changes broke the app. Thanks for pointing that out.

Reply:
Ok nevermind, if you reset the database (deleting the migrations) it works as expected.